internal/aws/aws.go

```diff
@@ -0,0 +1,27 @@
+package aws
+
+import (
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+)
+
+func NewSession(r string) *session.Session {
+	cfg := &aws.Config{}
+	if r != "" {
+		cfg.Region = aws.String(r)
+	}
+
+	endpoint := os.Getenv("AWS_ENDPOINT")
+	if endpoint != "" {
+		cfg.Endpoint = aws.String(endpoint)
+	}
```

Comment:
This will enable us to use a AWS compliant. I was debating between naming is `BISCUIT_AWS_ENDPOINT` or just `AWS_ENDPOINT`. I'm open to either but left it as is for now.